package.json

```diff
@@ -60,7 +60,7 @@
     "npm": "^2.14.0"
   },
   "scripts": {
-    "build": "gulp",
+    "build": "git clone -b fix\/tracker-ebola git://github.com/frictionlessdata/jsontableschema-js.git node_modules/jsontableschema && cd node_modules/jsontableschema && npm install && npm run build && gulp",
```

Comment:
it is run build inside jsontableschema folder with cloned repository. Anyway I will revert those changes, and back to hard coded lib in the code base